code/modules/uplink/uplink_items/spy_unique.dm

```diff
@@ -10,6 +10,7 @@
 	// Cost doesn't really matter since it's free, but it determines which loot pool it falls into.
 	// By default, these fall into easy-medium spy bounty loot pool
 	cost = SPY_LOWER_COST_THRESHOLD
+	uplink_item_flags = SYNDIE_ILLEGAL_TECH
```

Comment:
Also these blanket having illegal tech is totally an oversight on my part anyways

Reply:
Let me touch that up then while I'm in the ballpark, not a big deal